src/utils.js

```diff
@@ -0,0 +1,72 @@
+/**
+ * @license Copyright (c) 2003-2015, CKSource - Frederico Knabben. All rights reserved.
+ * For licensing, see LICENSE.md.
+ */
+
+'use strict';
+
+/**
+ * A utilities library.
+ *
+ * @class utils
+ * @singleton
+ */
+
+CKEDITOR.define( function() {
```

Comment:
Just to remember - we need to make a decision about https://github.com/ckeditor/ckeditor5-design/issues/13 and implement it sooner than later to have clear situation what tools and how can we use. It doesn't of course affect this review, but we should keep this in mind.